lang/cs/BUILD.md

```diff
@@ -24,7 +24,13 @@ Prerequisites
 -------------
 
   * Windows OS.
-  * [Building REEF Java](../java/BUILD.md).
+  * Apache maven
+  * Java version 7 or 8 (http://www.oracle.com/technetwork/java/javase/downloads)
+  * Protoc version 2.5 (https://github.com/google/protobuf/releases/tag/v2.5.0)
+  * Set M2_HOME environment variable to location of maven installation
+  * Set JAVA_HOME environment variable to java installation directory
+  * Set the location of the protoc executable in the windows path
```

Comment:
This is specific to the csharp build, unless we plan on porting csharp to linux which maybe the case I'm fine with windows path here